server/middleware/auth.js

```diff
@@ -18,4 +18,22 @@ const protect = (req, res, next) => {
   }
 };
 
+const protectSocket = (socket, next) => {
+  const { cookie } = socket.handshake.headers;
+  const token = cookie.split('token=')[1];
+
+  if (!token) {
+    next(new Error('No token, authorization denied'));
```

Comment:
I think you we should `throw` at this point `throw(new Error('No token, authorization denied'));`

Reply:
Thank you, changed as commented in commit.